pkg/rancher-desktop/components/Nav.vue

```diff
@@ -115,6 +125,9 @@ export default {
         },
       };
     },
+    isOnWhitelist(name: string): boolean {
+      return ['Epinio', 'Tachometer'].includes(name);
```

Comment:
This whitelist will not apply to new extensions. We can go ahead with this for the moment and keep this info in another way later.